losses/multiclass_losses.py

```diff
@@ -173,7 +174,7 @@ def loss(y_true: tf.Tensor, y_pred: tf.Tensor) -> tf.Tensor:
         :param y_pred: Predicted masks (tf.Tensor, shape=(<BATCH_SIZE>, <IMAGE_HEIGHT>, <IMAGE_WIDTH>, <N_CLASSES>))
         :return: Focal loss (tf.Tensor, shape=(None,))
         """
-        f_loss = -(class_weights * (1-y_pred)**gamma * y_true * K.log(y_pred))
+        f_loss = -(class_weights * (1-y_pred)**gamma * y_true * K.log(y_pred + epsilon))
```

Comment:
This makes sense. Maybe add a `K.clip(y_pred, tf.keras.backend.epsilon(), 1 - tf.keras.backend.epsilon())` instead of epsilon?